README.md

```diff
@@ -207,6 +252,7 @@ An example of using `fastify-swagger` with `static` mode enabled can be found [h
  | Option             | Default          | Description                                                                                                               |
  | ------------------ | ---------------- | ------------------------------------------------------------------------------------------------------------------------- |
  | exposeRoute        | false            | Exposes documentation route.                                                                                              |
+ | customCompiler        | false  | if it's `true` means fastify will support swagger open api specs (binary, byte, int32, int64) as a schema                                                                                            |
```

Comment:
```suggestion
 | customCompiler        | false  | If  `true` enable OpenAPI validation formats (binary, byte, int32, int64) as a schema                                                                                            |
```

Reply:
This option also needs to be moved to be above "exposeRoute" as the table is sorted alphabetically ascending.